content/events/2020-guadalajara/sponsor.md

```diff
@@ -3,64 +3,187 @@ Title = "Sponsor"
 Type = "event"
 Description = "Sponsor devopsdays Guadalajara 2020"
 +++
+<div class="container-fluid">
+  <div class="row justify-content-start">
+    <div class="col-md-9">
+      <div>
+      <h3></h3>
```

Comment:
Do we need an empty heading?

Reply:
oks used "<br>" instead of empty heading